### PR TITLE
add version to failover agent.py

### PR DESCRIPTION
**Pull request by mikeroup**
for #922

---
This change is [Reviewable](https://reviewable.io/reviews/volttron/volttron/924)

### CODE REVIEW

File: services/core/FailoverAgent/failover/agent.py [hunk 1]
```diff
@@ -67,6 +67,7 @@
 
 utils.setup_logging()
 _log = logging.getLogger(__name__)
+__version__ = '0.1'
```
Comment (craig8) [inline services/core/FailoverAgent/failover/agent.py]: Probably should be higher since this was out for a bit...Even though I doubt people have used it.

Comment (mikeroup) [inline services/core/FailoverAgent/failover/agent.py]: This is what Kyle said to do. Did you have a specific number in mind?

Comment (craig8) [inline services/core/FailoverAgent/failover/agent.py]: Kyle said that if would have thunk about it he would have had you bump it.  You can choose any number you like above 0.1.  I would however suggest 0.2.   But that's just me.  My guide would be http://semver.org/.

I might even be able to argue 0.1.1 as we changed the configuration interface.

Comment (mikeroup) [inline services/core/FailoverAgent/failover/agent.py]: I'll bump it to 0.2 when I add the alert publish.

**Pull request merged**